test/utils.py

```diff
@@ -217,6 +219,185 @@ def all_files_exist():
     return wait_event(timeout, all_files_exist, interval)
 
 
+class TtApp(object):
+    # Helper type to represent target. Commands (start, stop, etc.) treat it as following:
+    # None - no target at all (ex: tt start)
+    # Target(None) - app name only (ex: tt start some_app)
+    # Target('some_inst') - app:inst target (ex: tt start some_app:some_inst)
+    Target = namedtuple('Target', ['inst'])
```

Comment:
What are the benefits to use `namedtuple` construction? Especially for single field tuple.

Reply:
I've rearranged tests and dropped `Target`.